rhoai-2.22.1.md

```diff
@@ -84,6 +59,36 @@
     - quay.io/opendatahub/llama-stack-k8s-operator@sha256:f8f66ec02e7f59e510cb20eebfe0c1f6d41b62a9276c6e8e703d258f93d49119
     - quay.io/modh/must-gather@sha256:3b63f056d719ecc1ddd980c99272ade594440af03836bc780f8f394a56010d43
 
+# Optional images:
```

Comment:
Please consider renaming it to `Unsupported images` or `Deprecated images`

Reply:
Or PMs should be asked? Or doc writer?

Reply:
@jiridanek Maybe we could involve the doc writer to ensure consistency between what's written there and what's communicated here.